src/k8s-frameworkcontroller/deploy/stop.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+pushd $(dirname "$0") > /dev/null
+
+if kubectl get sts | grep -q "k8s-frameworkcontroller-sts"; then
+    kubectl delete sts k8s-frameworkcontroller-sts
```

Comment:
Wait until the sts is truely deleted before exit or just force delete it?

Reply:
Seems kubelet delete already default to --wait=true

Reply:
Seems kubectl cannot DeletePropagationForeground even in the master.
So, could you please delete the sts like delete framework:
https://github.com/microsoft/frameworkcontroller/blob/master/doc/user-manual.md#delete-framework
In addition, you need to wait until it is deleted, by looping (sleep, get, sleep, get)...